ReplayKitExample/BroadcastExtension/SampleHandler.swift

```diff
@@ -16,14 +16,15 @@ class SampleHandler: RPBroadcastSampleHandler, TVIRoomDelegate {
     var audioTrack: TVILocalAudioTrack?
     var videoSource: ReplayKitVideoSource?
     var screenTrack: TVILocalVideoTrack?
+    var disconnectSemaphore: DispatchSemaphore?
 
     var accessToken: String = "TWILIO_ACCESS_TOKEN"
     let accessTokenUrl = "http://127.0.0.1:5000/"
 
     static let kBroadcastSetupInfoRoomNameKey = "roomName"
 
     // In order to save memory, we request that our source downscale its output.
-    static let kDownScaledMaxWidthOrHeight = 640
+    static let kDownScaledMaxWidthOrHeight = 960
```

Comment:
Could also use 720, up for discussion. It doesn't seem necessary to reduce all the way to 640 when we can guarantee that the media engine is only handling the encoding and sending of outbound Tracks.

I still see image tearing in some cases regardless of scaling choices. Looking into this...